Comp2024/src/main/java/frc/robot/generated/TunerConstants.java

```diff
@@ -77,7 +76,7 @@ public class TunerConstants
   private static final int                          kFrontLeftDriveMotorId   = 1;
   private static final int                          kFrontLeftSteerMotorId   = 2;
   private static final int                          kFrontLeftEncoderId      = 3;
-  private static final double                       kFrontLeftEncoderOffset  = 0.414306640625;
```

Comment:
Not sure why these changes came in on your PR. I thought they would be on Hannah's branch. It's OK, but surprised me.